### PR TITLE
Apply phpcs-fixer

### PR DESCRIPTION
**Pull request by stefk**
This PR is the result of the execution of phpcs-fixer on the entire codebase. Should be merged before #141.

### CODE REVIEW

File: main/core/Command/CreatePluginCommand.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ protected function configure()
         $this->setDefinition(
             array(
                 new InputArgument('vendor', InputArgument::REQUIRED, 'The vendor name'),
-                new InputArgument('bundle', InputArgument::REQUIRED, 'The bundle name')
+                new InputArgument('bundle', InputArgument::REQUIRED, 'The bundle name'),
```
Comment (ngodfraind) [inline main/core/Command/CreatePluginCommand.php]: Do we really need a comma at the end of every element of every array ?

Comment (stefk) [inline main/core/Command/CreatePluginCommand.php]: Not a fan either, but that's the symfony convention.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by purplefish32** [comment]
LGTM

**Comment by LaurentGruber** [comment]
LGTM

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**